protobuf/election_contract.proto

```diff
@@ -164,6 +164,7 @@ message ElectorVote {
     repeated ElectionVotingRecord active_voting_records = 5;
     repeated ElectionVotingRecord withdrawn_votes_records = 6;
     bytes pubkey = 7;
+    aelf.Address address = 8;
```

Comment:
Why did we add `address` when it can be generated from `pubkey`? For performance reasons?